src/query.js

```diff
@@ -8,8 +8,8 @@ import { equals } from './utils.js'
  * This is a parquet-aware query engine that can read a subset of rows and columns.
  * Accepts optional filter object to filter the results and orderBy column name to sort the results.
  * Note that using orderBy may SIGNIFICANTLY increase the query time.
- *
- * @param {ParquetReadOptions & { filter?: ParquetQueryFilter, orderBy?: string }} options
+ * @import {BaseParquetReadOptions} from '../src/types.d.ts'
+ * @param {BaseParquetReadOptions & { filter?: ParquetQueryFilter, orderBy?: string }} options
```

Comment:
requires to fix a bug: we explicitly use parquetReadObject and expect it to return an array of objects (in main, if we call `parquetQuery({rowFormat: 'array', ...})` wit will throw an error at https://github.com/hyparam/hyparquet/blob/master/src/query.js#L60 and https://github.com/hyparam/hyparquet/blob/master/src/query.js#L86